admin/form-builder/assets/js/form-builder.js

```
@@ -914,31 +1047,263 @@
     // Mobile view menu toggle
-    $('#wpuf-form-builder').on('click', '#wpuf-toggle-field-options, #wpuf-toggle-show-form, .control-buttons .fa-pencil, .ui-draggable-handle', function() {
+    $('#wpuf-form-builder').on('click', '#wpuf-toggle-field-options, #wpuf-toggle-show-form, .field-buttons .fa-pencil, .ui-draggable-handle', function() {
         $('#wpuf-toggle-field-options').toggleClass('hide');
         $('#wpuf-toggle-show-form').toggleClass('show');
         $('#builder-form-fields').toggleClass('show');
     });
 
-    $('#wpuf_settings_posttype').on('change', function() {
+    $('select#post_type').on('change', function() {
         event.preventDefault();
+        var self = this;
```

Biome (1.9.4)

[error] This aliasing of this is unnecessary.

Arrow functions inherits `this` from their enclosing scope.
Safe fix: Use this instead of an alias.


(lint/complexity/noUselessThisAlias)